pkg/reconciler/service/service.go

```diff
@@ -294,6 +296,19 @@ func (c *Reconciler) reconcileRoute(ctx context.Context, service *v1.Service, ro
 	return c.client.ServingV1().Routes(service.Namespace).Update(existing)
 }
 
+func getIngressClass(ctx context.Context, s *v1.Service) string {
+	labels := s.GetLabels()
+	ingressClass := labels[networking.IngressClassAnnotationKey]
```

Comment:
I think we actually want to look at annotations here as that is what the Ingress owned by the Service will be looking at to determine ingressClass.

Reply:
Done.